### DISCUSSION TITLE
Deposit additional RING liquidity into HydraDX Omnipool

### DISCUSSION THREAD

**Discussion by spazcoin** [darwinia-network/darwinia#1489] [General]
I propose that the Darwinia treasury make an additional RING deposit into the HydraDX Omnipool to further increase on-chain liquidity. I propose adding **230M RING ($895k)** which is 28% of the current treasury. This would bring the RING treasury deposit in the Omnipool up to **$1M** or 2.3% of FDV.

<br>

The Darwinia treasury is currently depositing 25M RING into the Omnipool, approved by Darwinia voters in https://darwinia2.subsquare.io/referenda/2

With the initial listing deposit of 25M RING, you'll be able to trade 694k RING ($2,700) per trade before hitting 2% slippage. Recently in twitter discussions an investment firm mentioned they tried to buy 600k ASTR ($76k) but slippage was too high and they had to use a CEX instead.
https://twitter.com/Portela_Capital/status/1770811859525923256
DCA helps you break up large trades, but this reinforces the fact that we should continue to deepen our liquidity across the Polkadot ecosystem. Let’s bring those fees on-chain to Polkadot!

Another reason to increase RING liquidity on-chain is to support larger **money markets for RING**. HydraDX is launching a lending platform soon. Since large liquidation trades would be serviced by the Omnipool, increasing Omnipool liquidity would support a larger max cap for lending markets. This enables another use case for RING hodlers. I'm looking for other ways to help communities make use of their treasury-controlled funds to increase liquidity and usefulness of tokens across the entire Polkadot ecosystem. So I would separately encourage the RING treasury to consider later depositing RING liquidity ($200k?) into the HydraDX lending markets to enable larger borrowing positions. (but that would be a separate future Darwinia proposal)

The great thing about all of the proposals above is that they put to use “Treasury Owned Liquidity”.

- **earn fees**
- **add token use cases**
- **deepen liquidity**
- **no “listing fee” or reoccurring costs for market making or LM**
- **tokens aren’t “spent” and are still controlled by the Darwinia community**

**Q:** But how much in fees?
**A:** Omnipool stats show most treasuries earning 1.5-4% APR from swap fees. Deposits into lending would likely earn 3-8% once they are borrowed.

**Q:** But $895k from the treasury is a lot!!
**A:** Astar governance already passed a vote to [increase their ASTR deposit to $1M](https://forum.astar.network/t/deposit-additional-astr-into-hydradx-omnipool/6507). Bifrost has a passing proposal right now to [incraese their BNC deposit up to $1M](https://bifrost.subsquare.io/democracy/referenda/122). The Interlay community is moving towards boosting theirs to $1-1.2M. Phala community is considering my proposal to increase PHA deposit as well.
I’m certainly open to discussing deposit amounts other than 230M RING, but as I outline in this thread, I’m encouraging multiple teams to deposit $1M of liquidity to send a strong “we’re open for business” signal from a cohesive Polkadot community, with minimal risk for each treasury and demonstrating the power of XCM to control funds on other chains. I admit that 26% of the Darwinia treasury is A LOT to ask for, but to me this is a MUCH more productive use of the tokens than simply burning 400M RING.
https://twitter.com/spazvt/status/1770665393352225142

<br>

Final notes:

- This will have a side-effect of increasing the market capitalization of RING by $895k because tokens in the treasury acct aren’t counted towards MC but tokens in a DEX are.
- Disclosure: I am a $HDX token holder. I am not a HydraDX or Interlay team member.

**Comment by 2xJDubs** [comment]
I think this is a well thought out counter proposal to the current Treasury burn that is being voted on.  Doing this will help the exposure of RING as the HydraDX project gains exposure.  Currently, IMHO, HydraDX is the best DEX in the Polkadot ecosystem and supporting it with more RING liquidity will benefit both projects.

**Reply by 2xJDubs** [reply]
Based on discussions with the HydraDX team assets are limited to 2% weight cap of the total TVL of the Omnipool.  Currently that limits the additional RING that can be deposited to 125M to 150M.  This would still result in a treasury burn of 250M to 275M RING.

**Comment by SasoLithops** [comment]
Great points @spazcoin 

I wouldn't mind adding 100M more to Omniflix pool and burn 300M.

@hackfisher what do you think?
Proposal to burn is live, but we can still write new one if this would save 100M Ring in treasury.

**Reply by hackfisher** [reply]
Thanks for the proposals.

My perspective is that we should proceed gradually. Since there's already a 25M proposal that hasn't been fully executed and implemented, we should wait for that proposal to be completed and see the demand and trading volume of $RING on it. I haven't had much experience with HydraDX yet, so I may not be able to evaluate it at the moment. There are many DEXs in the market, and according to CoinGecko data, Uniswap on Ethereum and Arbitrum currently has the largest trading volume for $RING, with decent liquidity.

The magnitude of this proposal is significant, and my option is the proportion it occupies is too high, which increases the risk of market supply/demand. This is one of reasons mentioned "treasury size and governance risks" in the burn proposal.

Additionally, I don't think this counter-propose or is related to the burn proposal. I'd like to emphasize that even after burn, there will still be a considerable amount of $RING available in the Treasury to support such proposals, and this doesn't even take into account future Treasury income. I believe that continuously increasing liquidity based on sustained positive feedback from the community might be better than a one-time injection.

**Reply by SasoLithops** [reply]
Makes sense. It is just hard at times watching/thinking of so much value getting burned, although I approve this burning like I explained in previous Burning 400M Github discussion.

**Reply by 2xJDubs** [reply]
@hackfisher This makes a lot of sense and I can understand the desire to wait and see what trading volume looks like on HydraDX before committing more funds.  I appreciate your feedback here, thank you.

**Comment by JuneCribb** [comment]
Hii, 

I read your comment on [HydraDX Discussion ](https://hydradx.subsquare.io/posts/181#4) and I love your points. They are focused on how to help Ring gain exposure while helping the treasury control inflation without burning them all.

Why burn 400M Ring when you can provide enough liquidity on one of the best and most used DEX in the Polkadot ecosystem?

With @2xJDubs discussion with HDX team, adding 150M will amount to 175M(678k), with a deposit of 40M to Hydradx future lending platform leaving 185M to be burnt. 

@spazcoin ideas are productive from every angle and it won't hurt to try. On the bright side, more exposure could positively impact Darwinia and its utility(Msgport).

**Reply by 2xJDubs** [reply]
Hey June, we will need to keep the RING amount to $540K or less as that is the cap for the Omnipool right now with no coin being more than 2%.

**Comment by Laki389** [comment]
Hi @spazcoin , nice to see community members interested and involved in Github discussions.

IMHO, 25M $ring as a start is a good liquidity and further after a certain period we will have a good experience whether it is enough or it should be increased.
Based on the needs of the community and the situation.

I think the comparison to Astar and their $1M liquidity is inappropriate.
For Astar to have $1M liquidity on HydraDX, they don't need to put up 10.75% of their total supply.

p.s Thanks for your transparency, i.e. revealing to us that you have $hdx tokens.
I'm curious, do you have $ring tokens?

**Reply by 2xJDubs** [reply]
From talking to the HydraDX team it would be limited to around 100M RING so only 4.6% of the supply.   Also sounds like no other token would be able to exceed the 2% as well.

**Reply by Laki389** [reply]
Ring has liquidity in:

- **Uniswap V2: RING** 51,408,562
- **MEXC: Mexc.com** 28,923,448
- **Polygon (Matic)** 21,459,710
- **Gate.io** 18,050,584
- **Uniswap** V3 10,500,817
- **DeGate** 2,082,261
- **Arbitrum** (I can't find Arbitrum's address, but according to Coingecko it's about half that of Uniswap v2, so **about 20M+ liq.**
- **In other 2-3 CEX** whose addresses are not marked.

I forgot **Snowswap** with **1,749,506**.

Coming soon, **25M** on **HydraDX**.

In total, this gives almost **180M $ring liquidity**, excluding 2-3 CEXs that have unmarked addresses.

**Reply by spazcoin** [reply]
Total liquidity across 11 exchanges doesn't matter as much as the amount of liquidity depth in the top 1-3 exchanges. Users are going to look up the one exchange that has the most liquidity (or is in the ecosystem where they already have assets) and they're going to do their trades on that one exchange. From coingecko you can see that there's not a single exchange that you can trade $10k at a time without having >2% slippage (very likely MUCH more).

Note that HydraDX is the only exchange on the list that allows 1) single-sided unmanaged deposits (no CEX market maker fees, no need to sell RING for the other token in a Uni V2 token pair, no Uni V3 concentrated liquidity to manage) and the only one that allows 2) direct Darwinia Governance control of the funds (no trusted parties needed). This is why I've been championing treasury POL to HydraDX specifically (and why I bought HDX tokens).

To your other points:
- good point that Darwinia's MC is significantly lower than Astar's. Usually I've only been encouraging communities to deposit up to 2.5% of FDV or token supply (which would be 60M RING or $232k). But no other community is talking about burning tokens equivalent to 17% of their current token supply ($1.5M). 
- I was a top contributor in the Kusama crowdloan so I have quite a bit of CRAB, but not RING. You are correct that I haven't been very involved in Darwinia over time. I am involved in a lot of other projects in the Polkadot ecosystem, though.
- Regarding starting with the 25M deposit and wait to see how it performs, yes that had been my approach. But with the burn proposal I had to jump in now to propose a more productive alternative.

**Comment by aurexav** [comment]
https://darwinia2.subsquare.io/referenda/23
